site.py

```diff
@@ -0,0 +1,84 @@
+import os
+import subprocess
+import urllib
+import json
+import time
+import random
+
+from flask import Flask, render_template, url_for, request, redirect, send_from_directory
+
+
+app = Flask(__name__)
+
+# Confirm that a provided file has an acceptable extension.
+# In this case, '.csv' is the only allowed extension.
+def allowed_file(filename):
+    return ('.' in filename) and (filename.rsplit('.', 1)[1] in ALLOWED_EXTENSIONS)
+
+# Retrieve the files uploaded by the user, and if they are valid,
+#  store them in a new generated subfolder of tmp/.
+@app.route('/', methods=['GET', 'POST'])
+def upload_file():
+    if request.method == 'POST':
+        metadatafile = request.files['metadatafile']
+        datafile = request.files['datafile']
+        
+        if metadatafile and metadatafile.filename.endswith('.csv') \
+           and datafile and datafile.filename.endswith('.csv'):
+            print "Saving files..."
+            app.config['UPLOAD_FOLDER'] = 'tmp/'+str(random.randint(10000,99999))+'/'
```

Comment:
Let's normalize these names for consistency and so they can me mapped directly in the container. Define two constants in the module:

```python
METADATA_FILE = 'metadata.csv'
DATA_FILE = 'data.csv'
```

Then use those names when saving the files.

```python
metadatafile.save(os.path.join(folder, METADATA_FILE))
datafile.save(os.path.join(folder, DATA_FILE))
```